### PR TITLE
Print all live masters addresses to fsadmin report

### PR DESCRIPTION
**Pull request by ssz1997**
When doing command `fsadmin report`, not only the leader master address is showed, all addresses of live masters are now being printed for more complete information.

### CODE REVIEW

**Comment by LuQQiu** [comment]
@ssz1997 As you can see there is some error message saying that "Some checks were not successful". In this PR, the Java 8/11 unit tests failed. 
- Click the `Details` button of the failed Java 8/11 Unit tests
- Close the `Generate coverage report` and open the `Run tests`
- The console shows mvn failure results ` [ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M5:test (default-test) on project alluxio-shell: There are test failures.`
- Search `[ERROR] Tests run` and find
```
Error: 7.720 [ERROR] alluxio.cli.fsadmin.report.SummaryCommandTest.summary  Time elapsed: 1.227 s  <<< FAILURE!
java.lang.AssertionError: 

Expected: iterable containing ["Alluxio cluster summary: ", "    Master Address: testAddress", "    Web Port: 1231", "    Rpc Port: 8462", "    Started: 11-06-2005 01:59:03:122", "    Uptime: 143 day(s), 15 hour(s), 53 minute(s), and 32 second(s)", "    Version: testVersion", "    Safe Mode: false", "    Zookeeper Enabled: true", "    Zookeeper Addresses: ", "        [zookeeper_hostname1]:2181", "        [zookeeper_hostname2]:2181", "        [zookeeper_hostname3]:2181", "    Live Workers: 12", "    Lost Workers: 4", "    Total Capacity: 1309.92KB", "        Tier: MEM  Size: 1309.92KB", "        Tier: DOM  Size: 230.96KB", "        Tier: RAM  Size: 22.57KB", "    Used Capacity: 60.97KB", "        Tier: MEM  Size: 60.97KB", "        Tier: DOM  Size: 72.50KB", "        Tier: RAM  Size: 6.10KB", "    Free Capacity: 1248.94KB"]
     but: item 1: was "    Leader Master Address: testAddress"
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
```
Please fix the `SummaryCommandTest`.

**Comment by ssz1997** [comment]
@LuQQiu I'm only printing out the host of each NetworkAddress object, which I think is the same as the "address"?

File: shell/src/test/java/alluxio/cli/fsadmin/report/SummaryCommandTest.java [hunk 1]
```diff
@@ -57,6 +58,7 @@ public void prepareDependencies() throws IOException {
     mMetaMasterClient = mock(MetaMasterClient.class);
     MasterInfo masterInfo = MasterInfo.newBuilder()
         .setLeaderMasterAddress("testAddress")
+        .addMasterAddresses(NetAddress.newBuilder().setHost("testAddress2").build())
```
Comment (LuQQiu) [inline shell/src/test/java/alluxio/cli/fsadmin/report/SummaryCommandTest.java:61]: Can we test multiple master addresses, like 3 masters here including the leader master address?

File: shell/src/test/java/alluxio/cli/fsadmin/report/SummaryCommandTest.java [hunk 2]
```diff
@@ -115,7 +117,10 @@ private void checkIfOutputValid(String dateFormatPattern) {
     // Skip checking startTime which relies on system time zone
     String startTime =  CommonUtils.convertMsToDate(1131242343122L, dateFormatPattern);
     List<String> expectedOutput = Arrays.asList("Alluxio cluster summary: ",
-        "    Master Address: testAddress",
+        "    Leader Master Address: testAddress",
+        "    Number of Other Live Masters: 1",
```
Comment (LuQQiu) [inline shell/src/test/java/alluxio/cli/fsadmin/report/SummaryCommandTest.java:121]: An Alluxio cluster in fault-tolerant mode only has 3 or 5 masters, unlike workers which can be hundreds or thousands. If the information can be shown obviously in one line, we don't need to use two lines.

File: shell/src/test/java/alluxio/cli/fsadmin/report/SummaryCommandTest.java (hunk 1 above)
Comment (LuQQiu) [reply shell/src/test/java/alluxio/cli/fsadmin/report/SummaryCommandTest.java:61]: The master addresses should include the leader master address and are better to include three master net addresses. Three is a typical number to achieve fault tolerance.

**Review by LuQQiu — CHANGES_REQUESTED** [review]
Thanks for modifying the Summary command and the related tests. Left some comments

**Comment by LuQQiu** [comment]
@ssz1997 I ran the command locally the result is
```
 ~/alluxioFolder/alluxio  pr-now *151 ?2  bin/alluxio fsadmin report                                                                                                              255
Alluxio cluster summary:
    Number of Live Masters: 2
    Leader Master Address: localhost:19998
    Live Masters Addresses: localhost:19998, localhost
```
The conf is
```
alluxio.master.hostname=localhost
```
Please help 
1. Remove the number of live masters since in our use cases, the master number is usually no more than 5. The master number is obvious and no need to use an extra line to show this information.
2. Do not add the leader master address to live master addresses since the live master addresses already include the leader master address
3. Include the ports information in the live masters addresses information. (e.g. localhost:19998 instead of localhost)

File: shell/src/test/java/alluxio/cli/fsadmin/report/SummaryCommandTest.java [hunk 3]
```diff
@@ -57,6 +58,9 @@ public void prepareDependencies() throws IOException {
     mMetaMasterClient = mock(MetaMasterClient.class);
     MasterInfo masterInfo = MasterInfo.newBuilder()
         .setLeaderMasterAddress("testAddress")
```
Comment (LuQQiu) [inline shell/src/test/java/alluxio/cli/fsadmin/report/SummaryCommandTest.java:60]: add port information? like "testAddress:19998" to similar to local test results?

File: shell/src/test/java/alluxio/cli/fsadmin/report/SummaryCommandTest.java [hunk 4]
```diff
@@ -57,6 +58,9 @@ public void prepareDependencies() throws IOException {
...
+        .addMasterAddresses(NetAddress.newBuilder().setHost("testAddress").build())
```
Comment (LuQQiu) [inline shell/src/test/java/alluxio/cli/fsadmin/report/SummaryCommandTest.java:61]: set Port as well?

**Review by LuQQiu — CHANGES_REQUESTED** [review]
Left two minor comments, otherwise LGTM

**Review by LuQQiu — APPROVED** [review]
LGTM! Thanks for making those helpful changes!

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please!

**Pull request merged**